backend/src/user.ts

```diff
@@ -29,6 +29,18 @@ export class User {
 			return indices;
 		}, new Array<number>());
 	}
+	getIncorrect(): number[] {
+		return this.answers.reduce((indices, ans, i) => {
+			if (!ans.correct) indices.push(i);
```

Comment:
not completely certain, but does this count undefined answers? (I did not test it, just checking)

Reply:
That’s problematic. If it helps, we could remove “unanswered” and just use correct or wrong (total questions - correct) in the post-game statistics.

Reply:
i didn't pay attention to that but you're right, i don't think there's a way to track unanswered questions. i think jasmine's idea would be a good alternative for now.

Reply:
Ok, it's done on the frontend of this branch.

Reply:
> i didn't pay attention to that but you're right, i don't think there's a way to track unanswered questions. i think jasmine's idea would be a good alternative for now.

There is! if the answer is -1, the user never answered
the answerObject exists for every question, but some fields may remain blank